### ISSUE TITLE
OBS Recording Defaults to 5kBPS on 2nd recording after reconfigure. 

### ISSUE THREAD

**Issue by aza547** [aza547/wow-recorder#321]
This is a WTF one...
https://discord.com/channels/1004860808737591326/1072228542416429116

TL;DR:
- Recording has been bugged for months.
- Updating bitrate and recording something works fine.
- The next recording, the bitrate is reset to 5kbps (I think that's the default?)

I've no idea why. We use the same instance of OBS to record, the same recFactory variable on the Recorder class etc.

I tried printing out the recFactory.videoEncoder.settings on each start recording, the setting persists, but it's not passed down to OBS somehow. Despite that even the OBS logs show the correct bitrate, it's just the output file that's wrong. 

Adding the application of the videoEncoder to the `startOBS()` function seems to resolve it:
```
    this.obsRecordingFactory.videoEncoder.update({
      rate_control: 'VBR',
      bitrate: 1000 * this.cfg.get<number>('obsKBitRate'),
    });
```

**Comment by aza547** [comment]
Things I also tried:
- Upgrading to OSN 0.23.64, no luck.
- Upgrading to OSN 0.23.59do10 (no idea what this versioning scheme is), had to make changes to make recording work, still didn't fix it.

**Comment by aza547** [comment]
Things like this make me want to quit and become a gardener

**Comment by aza547** [comment]
Fix in https://github.com/aza547/wow-recorder/commit/ecbefbd77c0657ca6978ad6fba0901912709b5b2
